test/rxd/ecs/test_ecs_example.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from testutils import tol, compare_data
+
+@pytest.fixture
+def ecs_example(neuron_instance):
+    from math import pi
```

Comment:
I think we should generally favor imports at module level rather than function level